<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CadastaDialogBase</class>
 <widget class="QDialog" name="CadastaDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cadasta QGIS plugin</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>591</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="lineWidth">
    <number>5</number>
   </property>
   <property name="text">
    <string>Cadasta project creation wizard</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Organisation</string>
   </property>
  </widget>
  <widget class="QPushButton" name="get_organisations_button">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>110</y>
     <width>221</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Get available organisations ...</string>
   </property>
  </widget>
  <widget class="QComboBox" name="organisation_combo_box">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>231</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Project name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="project_name_input">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>160</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Project url</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="project_url_input">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>180</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>230</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>QGIS layer to upload</string>
   </property>
  </widget>
  <widget class="QComboBox" name="qgis_layer_combo_box">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>250</y>
     <width>211</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Project contact</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="project_contact_input">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="is_private">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Is this project private ?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>290</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Project Extents</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="layer_extents">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>310</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Use layer extents</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="current_cavas_extents">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>330</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Use current canvas extents</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Project description</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="project_description_input">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>400</y>
     <width>511</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="next_button">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>520</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Next -&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>530</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Step 1 &gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>530</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Step 2 &gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>530</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Step 3 &gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
